00:00:00.002478 main     VirtualBox COM Server 6.1.28 r147628 win.amd64 (Oct 18 2021 18:49:03) release log
00:00:00.002482 main     Log opened 2022-03-07T11:10:02.619433400Z
00:00:00.002482 main     Build Type: release
00:00:00.002486 main     OS Product: Windows 11
00:00:00.002487 main     OS Release: 10.0.22000
00:00:00.002488 main     OS Service Pack: 
00:00:00.078670 main     DMI Product Name: Inspiron 15 3511
00:00:00.092263 main     DMI Product Version: 
00:00:00.092272 main     Firmware type: UEFI
00:00:00.092562 main     Secure Boot: VERR_PRIVILEGE_NOT_HELD
00:00:00.092582 main     Host RAM: 7930MB (7.7GB) total, 2858MB (2.7GB) available
00:00:00.092587 main     Executable: C:\Program Files\Oracle\VirtualBox\VBoxSVC.exe
00:00:00.092588 main     Process ID: 22716
00:00:00.092588 main     Package type: WINDOWS_64BITS_GENERIC
00:00:00.284806          VirtualBox: object creation starts
00:00:00.285188          Home directory: 'C:\Users\reddy\.VirtualBox'
00:00:00.285375          Installed Drivers:
00:00:00.288823            C:\WINDOWS\system32\DRIVERS\VBoxNetLwf.sys (Version: 6.1.28.47628)
00:00:00.289632            C:\WINDOWS\system32\DRIVERS\VBoxUSBMon.sys (Version: 6.1.28.47628)
00:00:00.290216            C:\WINDOWS\system32\DRIVERS\VBoxDrv.sys (Version: 6.1.28.47628)
00:00:00.295184          Loading settings file "C:\Users\reddy\.VirtualBox\VirtualBox.xml" with version "1.12-windows"
00:00:00.300171          usbLibGetDevices: Starting USB device enumeration
00:00:00.304022          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&281396e5&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 6
00:00:00.304100          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&281396e5&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 10
00:00:00.305303          usbLibGetDevices: Found 4 USB devices, 0 captured
00:00:00.305362          USB: Unknown USB device detected (idVendor: 0x0bda, idProduct: 0xc829)
00:00:00.305406          USB: Unknown USB device detected (idVendor: 0x0c45, idProduct: 0x6730)
00:00:00.476729          netIfIsWireless: CreateFile on '\\.\{F44AD32E-5013-4F96-BB07-135880781B10}' failed with rcWin=2 (0x2) - ignoring
00:00:00.493504          HostDnsMonitor: initializing
00:00:00.508290          HostDnsMonitor: updating information
00:00:00.508296          HostDnsMonitor: unchanged
00:00:00.596595          VD: VDInit finished with VINF_SUCCESS
00:00:00.597729          Loading settings file "C:\Users\reddy\VirtualBox VMs\kalilinux\kalilinux.vbox" with version "1.16-windows"
00:00:00.615278          OCI: Local config file 'C:\Users\reddy\.VirtualBox\oci_config' does not exist
00:00:00.615362          OCI: Original config file 'C:\Users\reddy\.oci\config' does not exist
00:00:00.615363          OCI: Reading profiles finished with status ERROR_SUCCESS
00:00:00.615370          ExtPack: Created cloud provider 'OCI' (hrc=ERROR_SUCCESS)
00:00:00.615402          VirtualBox: object created
00:00:08.269522 main     VirtualBox: object deletion starts
00:00:08.271392 main     ERROR [COM]: aRC=VBOX_E_OBJECT_IN_USE (0x80bb000c) aIID={ad47ad09-787b-44ab-b343-a082a3f2dfb1} aComponent={MediumWrap} aText={Medium 'C:\Users\reddy\VirtualBox VMs\kalilinux\kalilinux.vdi' cannot be closed because it is still attached to 1 virtual machines}, preserve=false aResultDetail=0
00:00:08.285964 main     HostDnsMonitor: shutting down ...
00:00:08.286480 main     HostDnsMonitor: shut down
00:00:08.296098 Watcher  ERROR [COM]: aRC=E_ACCESSDENIED (0x80070005) aIID={d0a0163f-e254-4e5b-a1f2-011cf991c38d} aComponent={VirtualBoxWrap} aText={The object is not ready}, preserve=false aResultDetail=0
00:00:08.297567 main     VirtualBox: object deleted
